opam

```diff
@@ -1,8 +1,9 @@
 opam-version: "2.0"
 name: "ulex-camlp5"
-version: "1.2"
+version: "1.4"
```

Comment:
Why skip 1.3?

Reply:
You are right, no reason for that. I probably had already bumped it in my copy in the past and I forgot.

I have added a new commit to use 1.3.

Thanks!

Reply:
In the meantime I have released ulex-camlp5-1.3 in opam. Even the last version of the package (1.2) used to point to my fork, even if I would like to keep it in sync with your camlp5 branch.

Reply:
Note: I tested and the opam file can be changed to accept ocaml >= 4.14.1 too!